components/artist_page_cta/view.coffee

```diff
@@ -53,10 +53,18 @@ module.exports = class ArtistPageCTAView extends Backbone.View
       afterAuthPath: @afterAuthPath
     @$(".artist-page-cta-overlay__register input[name='name']").focus()
     @$('.artist-page-cta-overlay__close').on 'click', @closeOverlay
+    @disableScroll()
+
+  disableScroll: ->
+    $('#main-layout-container').addClass('artist-page-cta__no-scroll')
```

Comment:
Maybe it just shouldn't be BEM'ed (but I still like it in this component).

Maybe just an entirely new class `artist-page-cta-no-scroll` or something?